examples/with-mongodb/pages/index.tsx

```diff
@@ -1,8 +1,8 @@
 import Head from 'next/head'
 import clientPromise from '../lib/mongodb'
-import { InferGetServerSidePropsType } from 'next'
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from 'next'
 
-export async function getServerSideProps(context) {
+export async function getServerSideProps(context: GetServerSidePropsContext) {
```

Comment:
How about using the same pattern from the docs here:

https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props

Reply:
Sure, i made another commit following the same pattern. let me know your feedback.